src/soc/ibm/power9/istep_14_2.c

```diff
@@ -0,0 +1,51 @@
+#include <cpu/power/istep_14.h>
+#include <cpu/power/istep_13.h>
+
+static void thermalInit(void)
+{
+	for (size_t MCSIndex = 0; MCSIndex < MCS_PER_PROC; ++MCSIndex) {
+		for (size_t MCAIndex = 0; MCAIndex < MCA_PER_MCS; ++MCAIndex) {
+				mca_and_or(
+					mcs_ids[MCSIndex], MCAIndex, MCA_MBA_FARB3Q,
+					~PPC_BITMASK(0, 45),
+					PPC_BIT(10) | PPC_BIT(25) | PPC_BIT(37));
+		}
+		scom_and_for_chiplet(mcs_ids[MCSIndex], MCS_MCMODE0, PPC_BIT(21));
```

Comment:
`MCS_MCMODE0` is Nest register, different chiplet ID should be used for it. I already needed it in a couple of places (e.g. https://github.com/3mdeb/coreboot/blob/55545787bb5a7c338b34114d4a8c677bea5133a6/src/soc/ibm/power9/istep_13_8.c#L37), so maybe we should consider adding another table of IDs for mapping MCS chiplet ID to Nest chiplet ID.

Also, this is done just for functional MCSs.

Reply:
MCS is now checked if it's functional, and there is `mcs_to_nest` lookup table created. I have changed existing code to also use this lookup table.